elements/fullwidth_container/README.md

```diff
@@ -0,0 +1,53 @@
+Th full width container is the base element of the document. It can only be a direct child of the root, so can be placed only in the highest level of the document structure. The main feature of this element type is that the adjusted background color and/or image extends to the full width of the email client's view pane. 
+
+There is a new feature in the genertor, is it possible to add containers with different width (before that the full width container's width property was set to 600px wide). Now it's is possible to set every full width block individually wider than the usual 600px.
+
+```
+{
+	"root" : {
+	"children" : [ 
+		{
+			"type" : "FULLWIDTH_CONTAINER",
+			"background" : {
+				"image" : {
+					"src" : "https://example.png",
+					"repeat" : "no-repeat",
+					"position" : "center center"
+				},
+				"color" : "#60de55"
+			},
+			"width" : 600,
+			"children" : [ 
+				...
+			]
+
+			"hideOnMobile" : false,
+			"customData" : {},
+			"hideOnDesktop" : false
+		}
+	],
+	"type" : "ROOT"
+}
+```
+
+| Properties | Type | Values | Description |
+| --- | --- | --- | ---
+| background | object |  |[Read more](../../property-groups/background/README.md)
+| width | number | integer | the width of the content block (the email's body) can be set here, it is became a pixelated value |
```

Comment:
I would say: ... can be set here in pixels.

Also,  it's only the email's body if there is only one fw container in the doc.